unity-client/Assets/ABConverter/VisualTests.cs

```diff
@@ -211,49 +211,49 @@ public static GameObject[] LoadAndInstantiateAllAssetBundles()
             foreach (var hash in dependencyAbs)
             {
                 string path = abPath + hash;
-                var req = UnityWebRequestAssetBundle.GetAssetBundle(path);
 
                 if (SystemInfo.operatingSystemFamily == OperatingSystemFamily.MacOSX)
-                    req.url = req.url.Replace("http://localhost", "file:///");
+                    path = path.Replace("http://localhost", "file:///");
 
-                req.SendWebRequest();
```

Comment:
This is inside `ABConverter`. `ABConverter` has its own Environment with its own `IWebRequest` implementation with the editor mode concerns. We shouldn't be referencing the `WebRequestController` here. 

This is because we aim to move the `ABConverter` code in a separate repo in the future, so we should revert the changes of this file.

Reply:
Ups, I didn't know about it. You're right it is better not to touch anything related to `ABConverter` if we plan to move it out of the repo in the future. Thanks for the warning! Done in https://github.com/decentraland/explorer/pull/2256/commits/c30f9b5faffb9a87ccc94aef77c601a80dabc8d1

Reply:
Ty!